src/main/resources/avro/sequenceAnnotations.avdl

```diff
@@ -0,0 +1,227 @@
+@namespace("org.ga4gh.models")
+/**
+This protocol defines annotations on GA4GH genomic sequences It includes two
+types of annotations: continuous and discrete hierarchical.
+
+The discrete hierarchical annotations are derived from the Sequence Ontology
+(SO) and GFF3 work 
+
+   http://www.sequenceontology.org/gff3.shtml
+
+The goal is to be able to store annotations using the GFF3 and SO conceptual
+model, although there is not necessarly a one-to-one mapping in Avro records
+to GFF3 records.
+
+The minimum requirement is to be able to accurately represent the current
+state of the art annotation data and the full SO model.  Feature is the
+core generic record which corresponds to the a GFF3 record.
+*/
+protocol SequenceAnnotations {
+
+  import idl "common.avdl";
+  import idl "ontologies.avdl";
+
+  /**
+  Identifier from a public database
+  */
+  record ExternalIdentifier {
```

Comment:
ExternalIdentifier is a more general term than accession, as not
all databases refer to their identifiers as `accessions'.
However, conceptually it's the same thing.

Michael Baudis <notifications@github.com> writes:
> Suggestion (in metadata branch) is to use an array of accessions for all types
> of alternative object identifiers/URIs etc. https://github.com/ga4gh/schemas/
> blob/metadata/doc/source/metadata_intro.rst
> 
> —
> Reply to this email directly or view it on GitHub.*
>